### DISCUSSION TITLE
can you use tauri to build cli apps?

### DISCUSSION THREAD

**Discussion by Nisthar** [tauri-apps/tauri#3206] [Q&A] [answered]
can you use this library to build cli apps?

**Comment by amrbashir** [comment, marked as answer]
yes you can. Check https://tauri.studio/en/docs/guides/cli

**Reply by Nisthar** [reply]
![image](https://user-images.githubusercontent.com/11994157/149354023-83ff35b4-ad80-45e0-a8de-a2216877ec0f.png)

**Reply by amrbashir** [reply]
Sorry we just restructured the docs this morning. I updated the link above.
